ZupIT/ritchie-cli
# Adding ansible credentials and bugfix for 2.0

- What I did

Adoption has been working on formulas for the API Manager team. One of them requires the provisioning of some hcl files to the cluster, which is done with Ansible. So it is being added to the pool of credentials

There was a bug in when adding a new credential provider, it would not be listed because the default credentials were only added if there was no config file. Upon further investigation, I think this file is not rewritten on updates as well. So the solution was to check a diff of the default credentials vs the current config.json file

- How to verify it
Run rit set credential and check for the existence of ansible option

- Description for the changelog
Added ansible support to rit set credential
